contact-duplicate/.gitignore

```diff
@@ -3,4 +3,4 @@ target/
 dist/
 package-lock.json
 hubspot.config.yml
-.env*
+.env
```

Comment:
You can also use this:
```
.env*
!.env.sample
```

Then we still ignore all of their other .env files

Reply:
Resolving b/c @theJoestJoeToEverJoe beat me lol

Reply:
👍  Done: [ecc3d40](https://github.com/HubSpot/ui-extensions-examples/pull/25/commits/ecc3d40f5104ec7c9ccd767ace616ab2fa605b74)